CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.3] - 2021-04-05
```

Comment:
Inconsistent with the readme.

I agree that we can bump to 1.0.0, especially since the config format changed we should bump at least the minor version.

Reply:
Indeed was the config format change the reason for 1.x.x, because it's a breaking change.